core/capabilities/webapi/target/target_test.go

```diff
@@ -195,16 +200,7 @@ func TestCapability_Execute(t *testing.T) {
 
 		resp, err := th.capability.Execute(ctx, req)
 		require.NoError(t, err)
-		var values map[string]any
-		err = resp.Value.UnwrapTo(&values)
-		require.NoError(t, err)
-		fmt.Printf("values %+v", values)
-		statusCode, ok := values["statusCode"].(int64)
-		require.True(t, ok)
-		require.Equal(t, int64(200), statusCode)
-		respBody, ok := values["body"].([]byte)
-		require.True(t, ok)
-		require.Equal(t, "response body", string(respBody))
+		verifyResp(t, resp)
```

Comment:
There's a school of thinking that re-use in test assertions is a bad thing, that copying is better because it's more understandable without following into function.  Not sure how much I buy into that but it's a thought.